docs/README.md

```diff
@@ -60,58 +60,58 @@ The Media Communications Mesh (MCM) enables efficient, low-latency media transpo
 1. **Clone the repository**
 
    ```bash
-   $ git clone https://github.com/OpenVisualCloud/Media-Communications-Mesh.git
+   git clone https://github.com/OpenVisualCloud/Media-Communications-Mesh.git
    ```
 
 2. **Navigate to the Media-Communications-Mesh directory**
 
     ```bash
-    $ cd Media-Communications-Mesh
+    cd Media-Communications-Mesh
     ```
 
-3. **Install Dependencies**, choose between options `a)` or `b)`.
+3. **Install Dependencies**, choose between **OPTION A** or **OPTION B**.
 
-    a) Use all-in-one environment preparation script. The script is designed for Debian and was tested under `Ubuntu 20.04` and `Ubuntu 22.04`, kernel version 5.15 environments.
+    - **OPTION A:** Use all-in-one environment preparation script. The script is designed for Debian and was tested under `Ubuntu 20.04` and `Ubuntu 22.04`, kernel version 5.15 environments.
 
     To use this option run the following command:
 
     ```bash
     sudo ./scripts/setup_build_env.sh
     ```
 
-    b) The following method is universal and should work for most Linux OS distributions.
+    - **OPTION B:** The following method is universal and should work for most Linux OS distributions.
 
-    - XDP-tools with eBpf: Follow the simple guide [XDP-tools](https://github.com/xdp-project/xdp-tools.git) for installation instructions.
-    - libfabric: Follow the [libfabric from source](https://github.com/ofiwg/libfabric?tab=readme-ov-file#building-and-installing-libfabric-from-source) for installation instructions.
-    - MTL: Follow the [MTL setup guide](https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/doc/build.md) for installation instructions.
-    - E810 driver: Follow the [MTL NIC setup guide](https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/doc/e810.md) for installation instructions.
-    - gRPC: Refer to the [gRPC documentation](https://grpc.io/docs/languages/cpp/quickstart/) for installation instructions.
-    - Install required packages:
+        - XDP-tools with eBpf: Follow the simple guide [XDP-tools](https://github.com/xdp-project/xdp-tools.git) for installation instructions.
+        - libfabric: Follow the [libfabric from source](https://github.com/ofiwg/libfabric?tab=readme-ov-file#building-and-installing-libfabric-from-source) for installation instructions.
```

Comment:
please remove this point. libfabric is installed by `./scripts/setup_rdma_env.sh install` bellow. ( you keep link to source  and copy it there just for reference)

Reply:
Removed as requested.

Reply:
thanks